HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -247,5 +257,6 @@ def customizeHLTforCMSSW(process, menuType="GRun"):
 
     process = customizeHLTfor42514(process)
     process = customizeHLTfor42497(process)
+    process = customizeHLTfor42817and42860(process)
```

Comment:
```suggestion
    process = customizeHLTfor42869(process)
```

this PR name has to be used here.